test/surface/read-logged_in.test.ts

```diff
@@ -245,5 +245,50 @@ describe('Read-LoggedIn', () => {
     const result = await solidLogicBob.fetch(resourceUrl);
     expect(result.status).toEqual(403);
   });
-
+  it('Is disallowed to read .acl with Read', async () => {
+    const resourceUrl = `${testFolderUrl}9/test.txt`;
+    // This will do mkdir-p:
+    const creationResult =  await solidLogicAlice.fetch(resourceUrl, {
+      method: 'PUT',
+      body: '<#hello> <#linked> <#world> .',
+      headers: {
+        'Content-Type': 'text/turtle',
+        'If-None-Match': '*'
+      }
+    });
+    const aclDocUrl = await solidLogicAlice.findAclDocUrl(resourceUrl);
+    await solidLogicAlice.fetch(aclDocUrl, {
+      method: 'PUT',
+      body: makeBody('acl:Read', 'acl:Read', resourceUrl),
+      headers: {
+        'Content-Type': 'text/turtle',
+        // 'If-None-Match': '*' - work around a bug in some servers that don't support If-None-Match on ACL doc URLs
+      }
+    });
+    const result = await solidLogicBob.fetch(aclDocUrl);
+    expect(responseCodeGroup(result.status)).toEqual("4xx");
+  });
+  it('Is allowed to read .acl with Control+Read', async () => {
+    const resourceUrl = `${testFolderUrl}9/test.txt`;
+    // This will do mkdir-p:
+    const creationResult =  await solidLogicAlice.fetch(resourceUrl, {
+      method: 'PUT',
+      body: '<#hello> <#linked> <#world> .',
+      headers: {
+        'Content-Type': 'text/turtle',
+        'If-None-Match': '*'
+      }
+    });
+    const aclDocUrl = await solidLogicAlice.findAclDocUrl(resourceUrl);
+    await solidLogicAlice.fetch(aclDocUrl, {
+      method: 'PUT',
+      body: makeBody('acl:Control, acl:Read', 'acl:Control, acl:Read', resourceUrl),
+      headers: {
+        'Content-Type': 'text/turtle',
+        // 'If-None-Match': '*' - work around a bug in some servers that don't support If-None-Match on ACL doc URLs
```

Comment:
I'm thinking maybe we shouldn't call this a "bug". We can say "The URL of the ACL doc we are creating may differ only in the query part from a resource that already exists, so we can't expect If-None-Match to work for ACLs".

@acoburn what do you think?

Reply:
This comment came from the test I used as a template, removing this from here;